cmd/paratime/stats.go

```diff
@@ -180,9 +179,9 @@ func (s *runtimeStats) printStats() {
 	table.Render()
 }
 
-var runtimeStatsCmd = &cobra.Command{
-	Use:   "runtime-stats [<start-height> [<end-height>]]",
-	Short: "Show runtime statistics",
+var statsCmd = &cobra.Command{
+	Use:   "stats [<start-height> [<end-height>]]",
+	Short: "Show ParaTime statistics",
 	Args:  cobra.MaximumNArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := cliConfig.Global()
```

Comment:
Unrelated, but `oasis paratime stats --no-paratime` crashes due to a nil pointer dereference. Should probably return an error instead (before derefercing `npa.ParaTime` below).

Reply:
Or just change the flags so `--no-paratime` is not even available.

Reply:
Fixed by checking, if paratime was provided. Omitting --no-paratime won't work, because they may be no ParaTimes configured at all.